### ISSUE TITLE
Document broken methods

### ISSUE THREAD

**Issue by SiccarPoint** [landlab/landlab#281]
This issue attempts to maintain a list of old methods that no longer exist in Landlab, and their replacements.

If you find a method in your code that is now giving errors, record it here.

**Comment by SiccarPoint** [comment]
mappers.map_link_end_node_max_value_to_link() is now absent. Replaced by map_max_of_link_nodes_to_link.

**Comment by pohleric** [comment]
From the tutorial deAlmeida Overland Flow Component:
rmg.create_node_array_zeros('water_depth')

The function `create_node_array_zeros` is not available.
If instead `rmg.add_zeros('water_depth', at='node')` is used, the `of.gear_time_step(rmg)` from the overlandflow-object gives an error:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-26-97d81d334f99> in <module>()
      4     print(dt)
      5     # Now, we can generate overland flow.
----> 6     of.overland_flow(rmg, dt)
      7     # Increased elapsed time
      8     elapsed_time += dt

TypeError: overland_flow() takes exactly 1 argument (3 given)

I can only assume that the functions are not bahaving the same.

**Comment by mcflugen** [comment]
@pohleric: It looks like the example you are using is out-of-date (at least with the latest version of landlab from `master`). The `overland_flow` method no longer takes a `ModelGrid` as the first argument. Instead, it should be either `of.overland_flow(dt)` or just `of.overland_flow()`.

What version of landlab are you using?

**Comment by pohleric** [comment]
I will check this. I use version 0.5.0 installed via pip following the instructions with anaconda on http://landlab.github.io/#/ on a mac.

**Comment by jadams15** [comment]
Hi @pohleric, I think the overland flow tutorial is outdated as well. Apologies for that. Are you using the iPython notebook? or can you point me to the code you're using?

If you've updated your Landlab recently using pip, the tutorial will most likely not work at all. once I know what code you're using I can help you get up and running, as I'm the primary code author of that component. Keep us posted on your progress!

**Comment by pohleric** [comment]
Hi @jadams15 , I managed it. What @mcflugen pointed out was indeed the issue. The other things that are not as in the tutorial are the time steps as you can see in the code. I did use my own DEM, so it might be related to that...

I attached the notebook; had to pack it for github. Thanks for the help! I will certainly get more into it soon!
[overland_flow.ipynb.zip](https://github.com/landlab/landlab/files/235158/overland_flow.ipynb.zip)

**Comment by jadams15** [comment]
of course, keep us posted. if you plan to use Landlab some more, please let us know and we can add you to our Slack channel for landlab users. That, and these github issue trackers, are the easiest way to stay in contact with our dev team.

**Comment by pohleric** [comment]
Hi @jadams15 ,
I found some time again to dig a bit deeper and would certainly appreciate if you could add me to the mentioned "Slack channel". Are the people working on the hydrological components from ¿Washington? in there as well?

**Comment by jadams15** [comment]
Hi @pohleric, it appears someone has added you to the Slack channel. If you have any questions, feel free to contact us there!
